pySMART/smartctl.py

```diff
@@ -28,28 +28,55 @@
 
 
 class Smartctl:
-    def __init__(self, smartctl_path=SMARTCTL_PATH, options: List[str] = [], sudo: bool = False):
+    def __init__(self, smartctl_path=SMARTCTL_PATH, options: List[str] = [], sudo: Union[bool | List[str]] = False):
```

Comment:
I think you should change this into `Union[bool,List[str]]` in order to have compatibility with << python 3.10

see: https://docs.python.org/3.9/library/typing.html

Reply:
Good call, I can see I was thinking of that, but I used the pipe instead of a comma anyway :upside_down_face: